### PR TITLE
Release Cyanure on conda-forge

### PR DESCRIPTION
**Pull request by thomasryck**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/cyanure/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    - intel-openmp  # [win]
+  host:
+    - python
+    - libblas
+    - libcblas
+    - numpy >=1.18.0
+    - scipy
+    - scikit-learn
+    - pip
+  run:
```
Comment (synapticarbors) [inline recipes/cyanure/meta.yaml:33]: Please list all runtime requirements under `run`. I assume this includes `numpy`, `scipy` and `scikit-learn`. Also it looks like you are building against `numpy`, so please take a look at:
https://conda-forge.org/docs/maintainer/knowledge_base.html#building-against-numpy

File: recipes/cyanure/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    - python
+    
+
+test:
+  imports:
+    - cyanure
+  source_files:
+   - test/*.py
+  requires:
```
Comment (synapticarbors) [inline recipes/cyanure/meta.yaml:42]: You should remove all runtime dependencies of the library from here and just list `pytest`.

File: recipes/cyanure/meta.yaml (hunk 1 above)
Comment (thomasryck) [reply recipes/cyanure/meta.yaml:31]: I think I fixed the dependencies according to the documentation.

**Comment by thomasryck** [comment]
@synapticarbors Can you tell me if the package will be build against different python version ? And if it is not the case, what should I do to build against the different python ? I have difficulties finding documentation regarding this point.

**Comment by synapticarbors** [comment]
Also I'm happy to merge this assuming you don't have any other issues you want to try to address before then.

**Comment by thomasryck** [comment]
> I believe we are currently building binaries for python 3.7-3.10. I did a quick look through the documentation and didn't see it listed. What versions of python did you want to make sure were included?

I would have liked to also release on python 3.6,  if it is not possible I will change the recipe to change the minimum numpy version and to not skip python3.10.

Maybe my question was wrongly formulated but just to be sure all these python versions are build without "extra" configuration from the recipe ?

If it is simpler for you you can merge the request and I will rerender the recipe after the merge.

**Comment by synapticarbors** [comment]
I'm going to merge this for now. I'm actually not sure about how to change the configuration to build for python 3.6. You might ask in the general gitter channel for help https://conda-forge.org/docs/orga/getting-in-touch.html#gitter

**Pull request merged**